style_transfer/cli.py

```diff
@@ -246,7 +246,7 @@ def arg_info(arg):
     callback = Callback(st, args, image_type=image_type, web_interface=web_interface)
     atexit.register(callback.close)
 
-    url = f'http://{args.host}:{args.port}/'
+    url = f'http://localhost:{args.port}/'
```

Comment:
Another option would be to change the default `--host` to `localhost` as opposed to `0.0.0.0`